### ISSUE TITLE
s3backer --nbd not doing anything

### ISSUE THREAD

**Issue by Nikratio** [archiecobbs/s3backer#188]
Hi,

Maybe I'm doing something wrong, but if so then it would be great if s3backer could tell me what it is:

```sh
nikratio@vostro ~/i/s3backer (nbdkit_compat)> sudo ./s3backer --nbd --size=50G --blockSize=512k --region=eu-west-2  nikratio-backup/lb /dev/ndb0
s3backer: /dev/ndb0

nikratio@vostro ~/i/s3backer (nbdkit_compat) [1]> ./s3backer --help &| grep nbd
	s3backer --nbd [options] bucket[/subdir] /dev/nbdX
	--nbd                         Run as an NBD server instead of a FUSE filesystem


```

**Comment by archiecobbs** [comment]
Looks like there is a bug with process handling when _not_ using the `-f` flag.

**Comment by archiecobbs** [comment]
Should be fixed with ae7a763 - please give it a try.
